tests/Foundation/FoundationAuthenticationTest.php

```diff
@@ -29,7 +29,7 @@ class FoundationAuthenticationTest extends TestCase
     ];
 
     /**
-     * @var \Mockery
+     * @return Guard|\Mockery\LegacyMockInterface|\Mockery\MockInterface
```

Comment:
The `Guard` class here still needs a FQN.

Reply:
Just for learning, why it should be FQN?

Reply:
It's part of our coding standards.

Reply:
Do you know why?! Is it laravel standard or is coming from somewhere else?! just curious :smile: